next/messages/en.json

```diff
@@ -297,6 +297,10 @@
     "thisRegulationDoesntCancell": "This regulation doesn't cancel any regulation.",
     "linkToRegulationMessage": "Go to regulation",
     "relatedRegulations": "Related generally binding regulations",
+    "numberOf": {
+      "attachments": "{count} {count, plural, one {attachment} other {attachments}}",
+      "amendments": "{count} {count, plural, one {amendment} other {amendments}}"
+    },
```

Comment:
There is no need to nest this translation keys. We try to keep the json structure as simple as possible.

-->
```
"numberOfAttachments": ...
"numberOfAmendments": ...
```